### PR TITLE
Stock symbols and balance sheet

### PR DESCRIPTION
**Pull request by Bifabig**
- **Use Functional Components**
- **Implement lifecycle hooks**
- **Use store, actions, and reducers in React**
- **Implement Routing using React Router**
- **Style using CSS modules**
- **Write unit tests with React Testing Library and Jest**

### CODE REVIEW

File: src/components/Search.js
Comment (IngDavidTM) [inline src/components/Search.js:1]: - [ ] While you did a fantastic job implementing the required changes, kindly as mentioned by the previous reviewer, based on project requirements, you meant to use a Filter stateless component to filter data retrieved from the API, However, you didn't include the Filter stateless component. The code is focused on fetching balancesheet data using Redux and rendering it. There is no filtering functionality implemented in this component. I recommend you check the comment from my partner: https://github.com/Bifabig/react-redux-capstone/pull/1#discussion_r1248766552

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Bifabig  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bifabig** [comment]
> ## Hi @Bifabig 🙋‍♂️,
> Good job so far! You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!
> 
>  [ 68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966 ](https://camo.githubusercontent.com/78087f8205c5736aa2d7d71fc32f3ec42ca7302de1c46e79464a18f3fba32dfb/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966) [ ](https://camo.githubusercontent.com/78087f8205c5736aa2d7d71fc32f3ec42ca7302de1c46e79464a18f3fba32dfb/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f333151333645484a3743355a6f494d7737392f67697068792e676966)
> 
> ## Top highlight 🎯
> * The Pull Request follows the GitFlow ✅
> * The PR title and description are professional and descriptive ✅
> * Linters are working properly ✅
> 
> ## Suggested changes
> * [x]  _Check the comments under the review._ ⬇
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @IngDavidTM, thanks for taking the time to review my code… Kindly have a look at the changes I’ve made

**Review by shella12 — APPROVED** [review]
Hi @Bifabig, 

## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Filter component added ✔️ 
- Design is professional 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**